### ISSUE TITLE
Running two instances on the same machine?

### ISSUE THREAD

**Issue by envieme** [askrella/whatsapp-chatgpt#197]
I tried to run two instances of the app on the same machine. By cloning and building the app with different docker-compose in two different directories with different container and app names. But once the second build is started, the first hangs and the second never starts-up too. I tried using different session-data directory too. Why is this?

**Comment by Aj-Butt** [comment]
![Uyuyuuntitled](https://user-images.githubusercontent.com/124036011/232473491-335c81e4-5fa5-4c82-97ed-8492ae4307fb.png)
im running two  on same system just with a change of folder name, one with TTS mode on and 2nd with TTS mode off 
on sigle call i get both audio and text working fyn

**Comment by envieme** [comment]
I assume you are running it as a node app directly on your system? I am using docker. In docker compose I have tried setting up two containers with different names, networks, and volumes. Though they both build and run, they hang at some point in time and only one or neither runs.

The other thing I tried is have two folders with seperate docker-compose and then build them seperately and run each. But as soon as I run the second one both crash with this error:

node:internal/process/promises:289
whatschat1    |             triggerUncaughtException(err, true /* fromPromise */);
whatschat1    |             ^
whatschat1    | 
whatschat1    | [Error: ENOENT: no such file or directory, rename './db.tmp.json' -> './db.json'] {
whatschat1    |   errno: -2,
whatschat1    |   code: 'ENOENT',
whatschat1    |   syscall: 'rename',
whatschat1    |   path: './db.tmp.json',
whatschat1    |   dest: './db.json'
whatschat1    | }
whatschat1    | 
whatschat1    | Node.js v19.9.0

Am unable to understand how the same file is trying to be accessed by both runs in different containers with different volumes!
